utils/utils.py

```diff
@@ -78,6 +78,11 @@ def dump_log(config, metrics, split):
 def dump_top_k_prediction(config, classes, y_pred, k=100):
     """Dump top k predictions to the predict_out_path. The format of this file is:
     <label1>:<value1> <label2>:<value2> ...
+
+    Parameters:
+    classes (list): list of class names
+    y_pred (list): list of ndarray (shape: number of samples * number of classes)
```

Comment:
The input is not a list of ndarray if we use vstack.
But do we really need vstack?

Reply:
As discussed, I've added more commands on these functions. Please check for updates.